src/derp.h

```diff
@@ -35,9 +35,9 @@ struct __attribute__((packed)) derp_pkt {
 void derp_tick(struct wireguard_device *dev);
 
 // State transition functions
-err_t derp_send_http_upgrade_request(struct wireguard_device *dev);
-err_t derp_key_exchange(struct wireguard_device *dev, struct pbuf *buf);
-err_t derp_data_message(struct wireguard_device *dev, struct pbuf *buf, struct derp_pkt *packet);
+// err_t derp_send_http_upgrade_request(struct wireguard_device *dev);
```

Comment:
Should these be removed? Or they will be used later?

Reply:
I don't know. They weren't used right now.
@Jauler  ?

Reply:
I think we should remove all commented out code.